### PR TITLE
Update of JAXWS CXF bundles to use CXF 3.3.3

### PR DESCRIPTION
**Pull request by BerksanAtes**
for #8930

### CODE REVIEW

**Comment by BerksanAtes** [comment]
#build

**Comment by BerksanAtes** [comment]
#build

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.bnd [hunk 1]
```diff
@@ -8,19 +8,20 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-bindings-soap;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
-cxfVersion=3.3.0
+cxfVersion=3.3.3
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-rt-bindings-soap;${cxfVersion}}!/!*-INF/*
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.bnd]: Add exact here as well

Comment (neuwerk) [reply dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.3.2/bnd.bnd]: @${repo;org.apache.cxf:cxf-rt-bindings-soap;${cxfVersion};EXACT}!/!*-INF/*

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/bnd.bnd [hunk 2]
```diff
@@ -8,12 +8,12 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-frontend-jaxws;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
-cxfVersion=3.3.0
+cxfVersion=3.3.3
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-rt-frontend-jaxws;${cxfVersion}}!/!*-INF/*, \
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/bnd.bnd]: Add EXACT here as well

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/bnd.bnd [hunk 3]
```diff
@@ -8,19 +8,19 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-frontend-simple;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
-cxfVersion=3.3.0
+cxfVersion=3.3.3
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-rt-frontend-simple;${cxfVersion}}!/!*-INF/*
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.simple.3.2/bnd.bnd]: Add EXACT

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.management.3.2/bnd.bnd [hunk 4]
```diff
@@ -8,19 +8,19 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-management;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
-cxfVersion=3.3.0
+cxfVersion=3.3.3
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-rt-management;${cxfVersion}}!/!*-INF/*
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.management.3.2/bnd.bnd]: Add EXACT

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/bnd.bnd [hunk 5]
```diff
@@ -8,19 +8,19 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-ws-addr;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
-cxfVersion=3.3.0
+cxfVersion=3.3.3
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-rt-ws-addr;${cxfVersion}}!/!*-INF/*
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.addr.3.2/bnd.bnd]: Add EXACT

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/bnd.bnd [hunk 6]
```diff
@@ -8,19 +8,19 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-ws-policy;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
-cxfVersion=3.3.0
+cxfVersion=3.3.3
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-rt-ws-policy;${cxfVersion}}!/!*-INF/*
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/bnd.bnd]: Add EXACT

File: dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/bnd.bnd [hunk 7]
```diff
@@ -8,19 +8,19 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-wsdl;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
-cxfVersion=3.3.0
+cxfVersion=3.3.3
 
 -includeresource: \
    @${repo;org.apache.cxf:cxf-rt-wsdl;${cxfVersion}}!/!*-INF/*
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.rt.wsdl.3.2/bnd.bnd]: Add EXACT

File: dev/com.ibm.ws.org.apache.cxf.cxf.tools.validator.3.2/bnd.bnd [hunk 8]
```diff
@@ -8,13 +8,15 @@
...
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-tools-validator;3.3.3;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
 
 javac.source: 1.8
 javac.target: 1.8
 
+cxfVersion=3.3.3
+
 -includeresource: \
-   @${repo;org.apache.cxf:cxf-tools-validator;3.3.0}!/!META-INF/*
+   @${repo;org.apache.cxf:cxf-tools-validator;3.3.3}!/!META-INF/*
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.cxf.cxf.tools.validator.3.2/bnd.bnd]: Add EXACT

**Pull request merged**